cobbler.spec

```diff
@@ -350,6 +357,7 @@ sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler
 %license COPYING
 %doc AUTHORS.in README.md
 %config(noreplace) %{apache_webconfigdir}/cobbler_web.conf
+%config(noreplace) %{apache_webconfigdir}/cobbler.conf
```

Comment:
Why is this here? We already have it in the main cobbler package. Should we remove it from the main cobbler package?

Reply:
Yes we indeed do have this in there but when I remove it the package build fails on deb based systems if I reproduced this correctly. In this case I think it won't hurt.